app/_includes/hub-examples.html

```diff
@@ -134,7 +134,7 @@
 {% unless include.params.k8s_examples == false %}
 {% navtab Kubernetes %}
 
-First, create a [KongPlugin](https://github.com/Kong/kubernetes-ingress-controller/blob/main/docs/references/custom-resources.md#kongplugin)
+First, create a [KongPlugin](https://docs.konghq.com/kubernetes-ingress-controller/latest/references/custom-resources/#kongplugin)
```

Comment:
```suggestion
First, create a [KongPlugin](/kubernetes-ingress-controller/latest/references/custom-resources/#kongplugin)
```
Just changing this to a relative URL, that's the standard practice we use in Kong docs.